metadata-db-app/src/cmr/metadata_db/data/memory_db.clj

```diff
@@ -38,6 +39,22 @@
               (ingest-events/concept-delete-event tombstone)))
           (concat concepts tombstones))
 
+        ;; CMR-2520 Remove this case when asynchronous cascaded deletes are implemented.
```

Comment:
This code can be combined with the tag tombstone concept code above once `get-associations-for-tombstone` mutimethod is created as a result of refactoring in my other comment.